tranx/dataset2.py

```diff
@@ -55,11 +55,11 @@ def preprocess_conala_dataset(train_file, test_file, grammar_file, src_freq=3, c
     train_examples = preprocess_dataset(train_file, name='train',
                                         transition_system=transition_system, n_jobs=n_jobs)
 
-    # held out 200 examples for development
-    full_train_examples = train_examples[:]
+    # held out (1-split)*len(train_examples) examples for development
+    #full_train_examples = train_examples[:]
     # np.random.shuffle(train_examples)
-    dev_examples = train_examples[-2175:]
-    train_examples = train_examples[:-2175]
+    dev_examples = train_examples[int(split*len(train_examples)):]
```

Comment:
You better move such computations to a separate variable to make the code cleaner.
`n_train_examples = int(split * len(train_examples))`